Units/parser-python.r/type-statements.d/input-0.py

```diff
@@ -0,0 +1,9 @@
+# Derived from the comment in #4351 submitted by @mgedmin
+x = type(1)
+def f(x, a):
+    return 1
```

Comment:
I don't quite understand the reason for this one?

Reply:
"Garbage in, garbage out" is one of the mottos of ctags development.
I don't want to spend much time improving ctags for broken input.

Regarding the C language, I can easily determine whether an input is valid or not.
However, when it comes to Python, I am not so sure. Python is not special. I don't know much about most languages that ctags accepts.

So we have input validators for some languages. See misc/validators.
The validators determine whether an input is valid or not.

In this pull request, I added a validator for Python: https://github.com/universal-ctags/ctags/pull/4353/commits/d7a6356258c932e09a941f8d846c42a16c1d3c11.

The Python validator indicates that `y = f(type, ...)` on the next line is "invalid" if `f` is not defined. So I added `f` before the line.